.gitignore

```diff
@@ -0,0 +1,11 @@
+bundles/*
+discify/*
+dist/*
+node_modules/*
+.env.local
+ipfs-contractus
+package-lock.json.*
+package-lock.json
+docs/_build/*
+.DS_Store
```

Comment:
I would recommend you to put it to git global ignore 

> git config --global core.excludesfile '~/.gitignore'

see: https://stackoverflow.com/questions/7335420/global-git-ignore

Reply:
> I would recommend you to put it to git global ignore
> 
> > git config --global core.excludesfile '~/.gitignore'
> 
> see: https://stackoverflow.com/questions/7335420/global-git-ignore

I would say "also put it to git global ignore". It doesn't hurt, when it's in the project and keeps files from people without the global config out of the project. ^^